### ISSUE TITLE
Combine developer spec files into a single spec

### ISSUE THREAD

**Issue by peterfpeterson** [mantidproject/mantid#8679]
This issue was originally [TRAC 7834](http://trac.mantidproject.org/mantid/ticket/7834)

This ticket is blocked by : 
 - [X] [TRAC7784](http://trac.mantidproject.org/mantid/ticket/7784)

Supposedly we don't need to have separate mantid-developer spec files per operating system. Combine the new ones into a single spec file.

**Comment by stuartcampbell** [comment]
@peterfpeterson isn't this already done ?

**Comment by peterfpeterson** [comment]
Yes. See [mantid-developer.spec](https://github.com/mantidproject/mantid/blob/master/Code/Mantid/Build/dev-packages/rpm/mantid-developer/mantid-developer.spec)
